application/transaction.go

```diff
@@ -1,11 +1,15 @@
 package application
 
 import (
+	"time"
+
 	"github.com/almighty/almighty-core/log"
 
 	"github.com/pkg/errors"
 )
 
+const databaseTransactionTimeout = 6 * time.Minute
```

Comment:
Yes, it was just a first shoot to start a discussion. We should use an env var... i'll also add some tests